### PR TITLE
Query and update

### PR DESCRIPTION
**Pull request by Javivy**
In this milestone I:
✅Updated values of the database
✅DELETED items of the database inside a transaction and ROLLBACK the changes with a SAVEPOINT
✅Created queries making use of GROUP BY

‼️ Note: the screenshots are in the folder Screenshots

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -13,3 +13,43 @@ SELECT name, neutered FROM animals WHERE neutered='1';
 SELECT * FROM animals WHERE name!='Gabumon';
 
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+-- /*Project 2 QUERIES */
+
+BEGIN TRANSACTION;
+UPDATE animals SET species= 'unspecified';
+ROLLBACK;
```
Comment (Davidosky007) [inline queries.sql:21]: * Kindly verify that the change was made after updating animals species to `unspecified `
* Also `verify that the change was undone after the `rollback` query

File: queries.sql [hunk 2]
```diff
@@ -13,3 +13,43 @@ SELECT name, neutered FROM animals WHERE neutered='1';
...
+SELECT * FROM animals;
+
+BEGIN TRANSACTION;
+
+UPDATE animals SET species='digimon'WHERE name LIKE '%mon';
+UPDATE animals SET species='pokemon'WHERE species IS NULL;
+
+COMMIT TRANSACTION;
```
Comment (Davidosky007) [inline queries.sql:29]: * Kindly verify that change was made after updating the animals' table
* Also verify that the change persists after the commit

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @Javivy 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
* Good looking readme file  👍 
* Well written PR description 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @Javivy, 

### Good job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Clean and Organized code
:heavy_check_mark: Descriptive Readme file

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [OPTIONAL] As it is a [requirement ](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md)for this project, "`Remember to add screenshots with the results of your queries and with the results of your play with transactions to your Pull Request description.`", please add  screenshots of your queries results to the pull request description.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**